software/desktop-environments/en.md

```diff
@@ -30,4 +30,22 @@ Solus offers the MATE desktop environment / experience, tailored to advanced use
 
 ``` bash
 sudo eopkg install -c desktop.mate
-```
\ No newline at end of file
+```
+
+## Plasma
+
+Solus will offer the KDE Plasma desktop environment / experience in the next comming release. Plasma is a powerful and highly customizable desktop enviroment built with Qt and the KDE framework. It is reccomended to use the ISO but until then, you can try it right now by running the following command:
```

Comment:
- comming -> coming
- reccomended -> recommended

Spellchecking is your friend 😉

Reply:
Yeah, had it turned off by mistake in Atom :grin:

Reply:
I must say tht my spelling isn't usually this bad...

Reply:
Fixed now!